src/core/load_library.cpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2022 AntGroup CO., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+#ifdef _WIN32
+#include <string>
+#include <Windows.h>
+
+namespace lgraph {
+namespace dll {
+HINSTANCE LoadDynamicLibrary(const std::string& path) { return LoadLibrary(path.c_str()); }

```

Comment:
need else for non windows env?

also modify cmake?

Reply:
no. the implementation of these functions are in load_library.h for linux.
On Windows, they have to be moved to load_library.cpp because of name conflict in Windows.h.